submissions/kirill8210/popup-task/index.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>html_css_popup</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+            <a href="#"><img src="img/favicon.png" alt="favicon"></a>
+            <nav>
+                <ul class="menu">
+                    <li><a href="#">Mail</a></li>
+                    <li><a href="#">Images</a></li>
+                    <li>
+                        <input id="checkbox" class="checkbox" type="checkbox">
+                        <label for="checkbox">
+                            <img src="img/popup-button.png" alt="popup-button">
+                        </label>
+                        <ul class="popup">
+                            <li><a href="#"><img src="img/calendar.png" alt="calendar"><p>Calendar</p></a></li>
+                            <li><a href="#"><img src="img/mail.png" alt="mail"><p>Mail</p></a></li>
+                            <li><a href="#"><img src="img/news.png" alt="news"><p>News</p></a></li>
+                            <li><a href="#"><img src="img/maps.png" alt="maps"><p>Map</p></a></li>
+                            <li><a href="#"><img src="img/files.png" alt="files"><p>Files</p></a></li>
+                            <li><a href="#"><img src="img/contacts.png" alt="contacts"><p>Contacts</p></a></li>
+                            <li><a href="#"><img src="img/tanslation.png" alt="translation"><p>Translate</p></a></li>
+                            <li><a href="#"><img src="img/videos.png" alt="videos"><p>Videos</p></a></li>
+                            <li><a href="#"><img src="img/search.png" alt="search"><p>Search</p></a></li>
+                            <li><a href="#"><img src="img/photos.png" alt="photos"><p>Photo</p></a></li>
+                            <li><a href="#"><img src="img/apps.png" alt="apps"><p>Apps</p></a></li>
+                            <li><a href="#"><img src="img/documents.png" alt="documents"><p>Document</p></a></li>
+                            <li class="more">
+                                <input id="checkbox_more" class="checkbox_more" type="checkbox">
+                                <label for="checkbox_more">
+                                    <span>More</span>
```

Comment:
When `More` button is clicked, it is floats left for some reason, please, check.
Anyway, you don't need to show it after clicking

Reply:
ok

Reply:
1. Add `:focus` state
2. You don't need to show it after clicking

Reply:
ok